smithy-typescript-codegen/src/test/java/software/amazon/smithy/typescript/codegen/CommandGeneratorTest.java

```diff
@@ -14,46 +14,64 @@ public class CommandGeneratorTest {
     public void addsCommandSpecificPlugins() {
         testCommmandCodegen(
             "output-structure.smithy",
-            "getSerdePlugin(config, this.serialize, this.deserialize)"
+            new String[] {"getSerdePlugin(config, this.serialize, this.deserialize)"}
         );
     }
 
     @Test
     public void writesSerializer() {
         testCommmandCodegen(
             "output-structure.smithy",
-            ".ser("
+            new String[] {".ser("}
         );
     }
 
     @Test
     public void writesDeserializer() {
         testCommmandCodegen(
             "output-structure.smithy",
-        ".de("
+            new String[] {".de("}
         );
     }
 
-    private void testCommmandCodegen(String file, String expectedType) {
-        Model model = Model.assembler()
-                .addImport(getClass().getResource(file))
-                .assemble()
-                .unwrap();
+    @Test
+    public void writesOperationContextParamValues() {
+        testCommmandCodegen(
+            "endpointsV2/endpoints-operation-context-params.smithy",
+            new String[] {
+                "opContextParamIdentifier: { type: \"operationContextParams\", name: this.input.fooString }",
+                "opContextParamSubExpression: { type: \"operationContextParams\", name: this.input.fooObj.bar }",
+                "opContextParamWildcardExpressionList: { type: \"operationContextParams\", name: this.input.fooList.map(obj => obj) }",
```

Comment:
can this no-op map be avoided?

Reply:
Yes. Removed in [`9d7af21` (#1379)](https://github.com/smithy-lang/smithy-typescript/pull/1379/commits/9d7af2190d9ca3d16dfcf5b2ea7d834e1f39dfb7)